package/ios/RNSkia-iOS/SkiaDrawView.mm

```diff
@@ -139,6 +139,7 @@ - (void) handleTouches:(NSSet<UITouch*>*) touches withEvent:(UIEvent*) event {
       nextTouch.y = position.y;
       nextTouch.force = [touch force];    
       nextTouch.id = [touch hash];
+      nextTouch.isPencil = touch.type == 2 ? true : false; // Pencil vs Touch
```

Comment:
How about using [UITouchTypePencil](https://developer.apple.com/documentation/uikit/uitouchtype/uitouchtypepencil?language=objc) explicitly? Or define a corresponding enum/type in the TS side.

ref: https://developer.apple.com/documentation/uikit/uitouchtype?language=objc